docs/visualize/tsvb.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[TSVB]]
+== Visualizing your data with TSVB
```

Comment:
This title is ok, but it isn't consistent with the rest of the TOC.

Reply:
It is consistent with the new Timelion title. I'll update the rest when I reorg the Visualize section.

Reply:
Maybe we can find a way to use TSVB in the TOC, and "Visualizing your data with TSVB" as the page title. Otherwise, the TOC will be hard to scan if every title starts with "Visualizing..."